.dockerignore

```diff
@@ -0,0 +1,4 @@
+!.git/**
```

Comment:
If we're using `!`, but haven't configured any actual exclusions, do we need the `.dockerignore`?

Reply:
We were running into an issue with building the dockerfile as the `.git` directory was getting picked up, so this file will let us ignore it.